packages/tasit-sdk/src/helpers/DecentralandUtils.js

```diff
@@ -1,20 +1,27 @@
 import { ethers } from "ethers";
 import TasitContracts from "../../../tasit-contracts/dist";
-const { ropsten } = TasitContracts;
-const { MarketplaceProxy } = ropsten;
-const { address: MARKETPLACE_ADDRESS } = MarketplaceProxy;
+// ABI is the same among the networks. The ABI from local will work with tesnet.
+const { local } = TasitContracts;
+const { Marketplace } = local;
+const { abi: marketplaceABI } = Marketplace;
 
 // This util class is being used to fetch data from Decentraland Marketplace contract
 // Fetching data likely will be replaced with subgraph queries
 export default class DecentralandUtils {
   #marketplace;
-  #mana;
   #provider;
 
   constructor() {
-    // Using ethers.js because Tasit Action isn't working with ropsten
```

Comment:
Does this mean we can remove ethers as a dependency now too? I think we still may use it in ProviderFactory, so maybe not.

Reply:
That bug was fixed but `ethers.js` stills useful mainly because of event filter utils.

Reply:
Ah ok. Eventually we may reexpose that ourselves, right? Feel free to resolve this.

Reply:
I'm not sure but I think yes. For that specific case seems to me that this code will be replaced to `subgraph` queries.

Reply:
Resolving this.

Reply:
Ah yeah, makes sense